### PR TITLE
 Add deps file RH injection 

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -41,6 +43,7 @@
     <SystemNumericsVectorsPackageVersion>4.5.0-preview2-26313-01</SystemNumericsVectorsPackageVersion>
     <SystemRuntimeCompilerServicesUnsafePackageVersion>4.5.0-preview2-26313-01</SystemRuntimeCompilerServicesUnsafePackageVersion>
     <SystemSecurityPrincipalWindowsPackageVersion>4.5.0-preview2-26313-01</SystemSecurityPrincipalWindowsPackageVersion>
+    <Tooling_NewtonsoftJsonPackageVersion>9.0.1</Tooling_NewtonsoftJsonPackageVersion>
```
Comment (jkotalik) [inline build/dependencies.props]: Why Tooling prefix? Also 11.0.1

**Comment by jkotalik** [comment]
Also appveyor is failing.

**Comment by jkotalik** [comment]
@pakrym I merged these changes with #732, that cool?

**Comment by pakrym** [comment]
Yep.

**Comment by pakrym** [comment]
Or let's just merge this if it works. You change would be smaller and don't have less related changes.

**Pull request merged**